src/items/use-declarations.md

```diff
@@ -27,16 +24,17 @@ and [blocks], usually at the top.
 
 Use declarations support a number of convenient shortcuts:
 
-* Simultaneously binding a list of paths differing only in their final element,
-  using the glob-like brace syntax `use a::b::{c,d,e,f};`
-* Simultaneously binding a list of paths differing only in their final element
-  and their immediate parent module, using the `self` keyword, such as `use
-  a::b::{self, c, d};`
+* Simultaneously binding a list of paths with a common prefix, using the
+  glob-like brace syntax `use a::b::{c, d, e::f, g::h::i};`
+* Simultaneously binding a list of paths with a common prefix and their common
+  parent module, using the `self` keyword, such as `use a::b::{self, c, d::e};`
 * Rebinding the target name as a new local name, using the syntax `use p::q::r
-  as x;`. This can also be used with the last two features: `use a::b::{self as
-  ab, c as abc}`.
+  as x;`. This can also be used with the last two features:
+  `use a::b::{self as ab, c as abc}`.
 * Binding all paths matching a given prefix, using the asterisk wildcard syntax
-  `use a::b::*;`
+  `use a::b::*;`.
+* Nesting groups of the previous features multiple times, such as
+  `use a::b::{self as ab, c d::{*, e::f}};`
```

Comment:
missing a `::` or `,` between `c` and `d`.

Reply:
Fixed!